app/controllers/bulk_updates_controller.rb

```diff
@@ -167,4 +190,19 @@ def ensure_correct_data_center_update_value(task)
       task['update-type'] = 'FIND_AND_UPDATE_HOME_PAGE_URL'
     end
   end
+
+  def reconstruct_failed_task(params)
+    # bulk update create action failed, so we need to reconstruct the task to repopulate the form
+
+    reconstructed_bulk_update_object = {
+      name:           params[:name],
+      update_field:   params.fetch(:update_field, '').downcase,
+      update_type:    params[:update_type],
+      find_value:     params[:find_value],
+      update_value:   params[:update_value]
+    }
+
+    Rails.logger.info "!!!!**** !!!! **** reconstructed task #{reconstructed_bulk_update_object.inspect}"
```

Comment:
probably don't need the !!!!**** in the logs

Reply:
yeah, I'm removing the line